gradlew.bat

```diff
@@ -64,29 +64,14 @@ echo location of your Java installation.
 
 goto fail
 
-:init
-@rem Get command-line arguments, handling Windows variants
-
-if not "%OS%" == "Windows_NT" goto win9xME_args

```

Comment:
Wow it has `goto` statement, and tries to support Windows ME! Who is still using that OS...